### PR TITLE
feat: add token in api request header for private repos

### PR DESCRIPTION
**Pull request by wu-cl**
`https://api.github.com/repos/xx/xx/pulls/xx/commits` is not accessible in private repos without access token

### CODE REVIEW

**Comment by wu-cl** [comment]
@adrians5j

**Comment by pallabpain** [comment]
https://github.com/webiny/action-conventional-commits/issues/8#issuecomment-1495366839

^ facing this issue in one of my private repos. Can we merge this PR?

**Comment by Zheer09** [comment]
Same here

**Comment by adrians5j** [comment]
Sorry for delay, was on a holiday. Will check this tomorrow.
Thanks!

File: action.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 name: "Conventional Commits"
 description: "Ensures that all commit messages are following the conventional-commits standard."
+inputs:
+  GITHUB_TOKEN:
+    description: 'GitHub token'
+    required: true
```
Comment (adrians5j) [inline action.yml:6]: Hm, do we want to mark this as required? 🤔 

I mean, the action worked without it, so I guess we should make this optional. Can we do that?

**Review by adrians5j — CHANGES_REQUESTED** [review]
Left a comment. Thanks!

File: action.yml (hunk 1 above)
Comment (wu-cl) [reply action.yml]: set required = false, Pls check it out.

**Review by pallabpain — APPROVED** [review]
Looks good.

**Review by adrians5j — APPROVED** [review]

**Pull request merged**